packages/react/src/lazy.ts

```diff
@@ -108,3 +109,126 @@ const createLazy =
 export const lazy = Object.assign(createLazy({}), {
   create: createLazy,
 })
+
+export interface ReloadOnErrorStorage {
+  getItem: (key: string) => string | null
+  setItem: (key: string, value: string) => void
+  removeItem: (key: string) => void
+}
+
+export interface ReloadOnErrorOptions extends LazyOptions {
```

Comment:
I thought this shouldn't be exported

```suggestion
interface ReloadOnErrorOptions extends LazyOptions {
```